interface/utils.py

```diff
@@ -57,3 +47,30 @@ def random_code(length, vocabulary=vocabulary_64):
 def is_true(value):
     text = (value or '').lower().strip()
     return text in ['1', 'yes', 'true', 'on', 'enabled']
+
+    
+def get_url_base(link):
+    m = re.match(r'https://github.com/(?P<org>[^/]+)/(?P<repo>[^/]+)/?$',
+                    link.assignment.repo_url)
+    url_base = ('https://raw.githubusercontent.com/'
+                '{0}/{1}/'.format(*list(m.groups())))
+    return url_base
+
+def get_config_url(link):
+    url_base = get_url_base(link)
+    config_url = urljoin(url_base,
+                             f'{link.assignment.repo_branch}/checker.sh')
+    return config_url
+
+def get_config_data(link):
+    url_base = get_url_base(link)
+    config_data = requests.get(
+                    urljoin(
+                        url_base,
+                        f'{link.assignment.repo_branch}/config.ini',
+                    )
+                  )
+    return config_data
+
+
+
```

Comment:
Too many new lines at the end of the file. Should be just one, right after `return config_data`. Also, a new line just before `return` would look really nice.